Jenkinsfile_CNP

```diff
@@ -92,7 +92,7 @@ withPipeline(type, product, component) {
   }
 
   before('functionalTest:aat') {
-    env.PRL_CITIZEN_URL = "https://prl-citizen-frontend.aat.platform.hmcts.net/"
```

Comment:
this must be the url - "https://prl-citizen-frontend-staging.service.core-compute-aat.internal/"